src/database/migrations/2018_07_05_131300_add_message_id_column.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddMessageIdColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('email_log', function (Blueprint $table) {
+            $table->string('messageId', 32)->nullable()->default(null);
```

Comment:
I think `message_id` might be a better name. There is no set precedent for snake case vs camel case at the table level, but the table name being snake case I think it follows that the columns should be snake case.

Reply:
Here is the migration code 

    up() {
       Schema::table('email_log', function (Blueprint $table) {
        $table->renameColumn('messageId', 'message_id');
      });
    }
    down() {
       Schema::table('email_log', function (Blueprint $table) {
        $table->renameColumn('message_id', 'messageId');
      });
    }


@baileylo Do you maybe know do I need to change the indexes also or would this suffice?